packages/grafana-data/src/types/navModel.ts

```diff
@@ -21,6 +21,7 @@ export interface NavModelItem {
   menuItemType?: NavMenuItemType;
   highlightText?: string;
   tabSuffix?: ComponentType<{ className?: string }>;
+  showInNavBar?: boolean;
```

Comment:
think we've change to `hideFromNavbar` in the backend. doesn't have to change anything in this PR, we can swap it later 😄